test/Microsoft.Health.Dicom.Web.Tests.E2E/Common/DicomTagsManager.cs

```diff
@@ -98,5 +98,20 @@ public async Task<GetExtendedQueryTagEntry> UpdateExtendedQueryTagAsync(string t
             var response = await _dicomWebClient.UpdateExtendedQueryTagAsync(tagPath, newValue, cancellationToken);
             return await response.GetValueAsync();
         }
+
+        public async Task<bool> DeleteExtendedQueryTagAsync(string tagPath, CancellationToken cancellationToken = default)
```

Comment:
I'm thinking if it's good to keep consistent with other functions in this class, and move specific login into test class. Because we not always want to eat the exception.

```csharp
 public async Task<bool> DeleteExtendedQueryTagAsync(string tagPath, CancellationToken cancellationToken = default)
        {
            EnsureArg.IsNotNull(tagPath, nameof(tagPath));
            await _dicomWebClient.DeleteExtendedQueryTagAsync(tagPath, cancellationToken);
        }
```